src/module/crud.js

```diff
@@ -1,19 +1,22 @@
-import { add, del, getLen } from './meme-object.js';
+import {
+  add, del, getLen, updateindex,
+} from './meme-object.js';
 import dom from './add-Dom-elem.js';
 
 const inputText = document.getElementById('add-text');
 
 const btnInsert = () => {
   add(inputText.value);
-  inputText.value = '';
   dom.creatElements(inputText.value, getLen());
+  inputText.value = '';
 };
 
 const btnDelete = () => {
   const checkboxlist = document.querySelectorAll('.checkbox');
   checkboxlist.forEach((element) => {
     if (element.checked === true) del(element.getAttribute('id'));
   });
+  updateindex();
 };
```

Comment:
- [ ] Great job implementing the `delete all completed tasks` but this functionality is supposed to be built into your upcoming project. For this project, you should implement functionality that allows users to delete individual tasks from the list, please make sure you application has this functionality, and please keep in mind Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.
- You can use a delete icons near each task that can be used by users to delete the tasks from the list.